test/basic.test.js

```diff
@@ -89,16 +89,35 @@ function levelTest (name, level) {
     const stream = sink()
     const instance = pino(stream)
     instance.level = name
-    instance[name]('hello world')
-    check(is, await once(stream, 'data'), level, 'hello world')
+    const m = 'hello world'
+    equal(instance[name](m), m)
```

Comment:
use the locally scoped `is`

Reply:
same for the rest of the usage of `equal`

Reply:
✅